flink-ml-core/src/main/resources/META_INF/licenses/MIT.license

```diff
@@ -0,0 +1,7 @@
+Copyright <YEAR> <COPYRIGHT HOLDER>
```

Comment:
Follow the pattern of the existing license files added in Flink, we need to copy the LICENSE from the `dev.ludovic.netlib:blas` repo and name this file `LICENSE.blas`.

Reply:
Thanks for pointing this out. This issue has been fixed.